frontend-web/webclient/app/Applications/Jobs/View.tsx

```diff
@@ -664,15 +664,57 @@ const InfoCard: React.FunctionComponent<{
 
 const RunningText: React.FunctionComponent<{job: Job}> = ({job}) => {
     return <>
-        <Heading.h2>
-            {!job.specification.name ? "Your job" : (<><i>{job.specification.name}</i></>)} is now running
-        </Heading.h2>
-        <Heading.h3>
-            <i>
-                {job.specification.resolvedApplication?.metadata?.title ?? job.specification.application.name}
-                {" "}v{job.specification.application.version}
-            </i>
-        </Heading.h3>
+        <Flex justifyContent={"space-between"}>
```

Comment:
We should probably try to extract the common part into its own component if possible.